heat/scripts/mount_share.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+import os,json,re,requests
```

Comment:
could you please import all the packages in different lines to follow PEP style guide?
https://www.python.org/dev/peps/pep-0008/#imports